### PR TITLE
snappi unit test map

### PR DESCRIPTION
**Pull request by Rangababu-R**
snappi unit test map

### CODE REVIEW

File: tests/ipv4/test_ip_priority_field.py [hunk 1]
```diff
@@ -12,26 +8,19 @@ def test_counter_ip_dscp(api, b2b_raw_config, utils):
...
     ]
-
     for i, p in enumerate(phb):
-        dscp = flow_ipv4.Dscp(
-            phb=flow.Pattern(getattr(flow_ipv4.Dscp, p))
-        )
-        prio = flow_ipv4.Priority(dscp)
-        f.packet[1].ipv4.priority = prio
-        utils.apply_config(api, b2b_raw_config)
+        ipv4.priority.choice = 'dscp'
```
Comment (ashutshkumr) [inline tests/ipv4/test_ip_priority_field.py:21]: we don't need to set a choice here.

File: tests/ipv4/test_ip_priority_field.py [hunk 2]
```diff
@@ -55,11 +43,13 @@ def test_ip_priority_tos(api, b2b_raw_config, utils):
     """
 
     f = b2b_raw_config.flows[0]
-    f.packet = [
-        flow.Header(flow.Ethernet()),
-        flow.Header(flow.Ipv4())
-    ]
-    utils.apply_config(api, b2b_raw_config)
+    # import snappi
```
Comment (ashutshkumr) [inline tests/ipv4/test_ip_priority_field.py:46]: these comments are not needed anymore i suppose, so remove them

**Review by ashutshkumr — CHANGES_REQUESTED** [review]
please address minor changes

File: tests/ipv4/test_ip_priority_field.py (hunk 1 above)
Comment (Rangababu-R) [reply tests/ipv4/test_ip_priority_field.py:21]: directly setting phb value throwing attribute error
>>> print(i)
{}
>>> i.priority.dhcp.phb.value = 10
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'FlowIpv4Priority' object has no attribute 'dhcp'
>>>

File: tests/ipv4/test_ip_priority_field.py (hunk 2 above)
Comment (Rangababu-R) [reply tests/ipv4/test_ip_priority_field.py:46]: yes missed to remove it.

File: tests/ipv4/test_ip_priority_field.py (hunk 1 above)
Comment (ashutshkumr) [reply tests/ipv4/test_ip_priority_field.py:21]: it should be `dscp`.

**Review by ashutshkumr — APPROVED** [review]

**Pull request merged**